job_runner/reporting/file.py

```diff
@@ -83,10 +86,6 @@ def report(self, msg):
                     msg.timestamp
                 )
                 self._dump_ok_file()
-            else:
-                # this has already been handled by earlier event
-                pass
-
```

Comment:
You need to keep some form of this. It's explaining why the failure isn't handled.